### ISSUE TITLE
[Feature Request] Provide workaround to allow building source code on Windows

### ISSUE THREAD

**Issue by AluminiumTech** [keeweb/keeweb#967]
Hi, 
I'm having issues building KeeWeb on Windows 10. I've read on the readme that building KeeWeb requires macOS as it builds dmg.

For those of us on Windows 10 who want to contribute, is there a way to work around this issue and disable building dmg?

Upon cloning the repository and trying to get it to run, following the build instructions it shows this:
![image](https://user-images.githubusercontent.com/14107316/43543522-2a349212-95c8-11e8-9a58-fda8aaaefe35.png)

I would really appreciate a way for Windows users to build this app and not have issues. 

Thanks.

**Comment by antelle** [comment]
Hi! It looks like you need to checkout `develop` branch, this is fixed there.

**Comment by AluminiumTech** [comment]
@antelle  I'm having issues with the develop branch.

It's not detecting the Python installation that definitely exists.

![image](https://user-images.githubusercontent.com/14107316/43611392-e310bd78-96a0-11e8-9b6c-aa462cc81dfc.png)

**Comment by AluminiumTech** [comment]
> I guess, you need to install python into this path

No, I'm saying that I already have python installed to that directory but it's not detecting it.

**Comment by antelle** [comment]
could be 64/32 bit version mismatch or something like this, it says it can’t find it there

**Comment by AluminiumTech** [comment]
> could be 64/32 bit version mismatch or something like this, it says it can’t find it there

I have the 64 bit version and that's the directory for the 64 bit version so I don't know.

**Comment by antelle** [comment]
I don’t know, node-sass is a module that should work on windows, try to google how to solve it.
keytar won’t build, that’s ok, building will work withiut it.

**Comment by Coises** [comment]
AluminumTech,

I’ve been building from the Develop branch on Windows 7, but it took a bit of fussing around. I’m sure I do not have Python installed anywhere, unless it got installed as a dependency within KeeWeb. I didn’t write down what I did, but I think it was this:

* Did the git and GitHub stuff to get KeeWeb into a folder managed by git.

* Installed [node.js](https://nodejs.org/en/), LTS version. (I installed 8.11.3; it looks like they’re on 8.11.4 now.)

* Opened a command line in the KeeWeb folder and ran `grunt --skip-sign`.

* Ran `npm install `_whatever-the-error-message-said-was-missing_. (First time was grunt itself.)

* Repeated the above two steps until it ran.

That got me a web app, but I wanted an Electron app. Eventually, I inserted this just before the last non-blank line in `Gruntfile.js`:

    grunt.registerTask('win32', 'Build unsigned web app (in dist) plus unsigned program folders (in tmp) for testing on Windows', [
        'gitinfo',
        'clean',
        'eslint',
        'copy:html',
        'copy:favicon',
        'copy:icons',
        'copy:manifest',
        'copy:fonts',
        'webpack',
        'uglify',
        'sass',
        'postcss',
        'inline',
        'htmlmin',
        'string-replace:manifest-html',
        'string-replace:manifest',
        'copy:dist-icons',
        'copy:dist-manifest',
        'gitinfo',
        'clean:desktop',
        'build-desktop-app-content',
        'electron:win32',
        'copy:desktop-windows-helper-ia32',
        'copy:desktop-windows-helper-x64'
    ]);

Now, when I have changes to test, I run `grunt win32`.
    
That leaves a folder, `KeeWeb-win32-x64`, in `tmp\desktop`, from which KeeWeb can be run by double-clicking the exe file. I haven’t tried to do anything with `grunt dev`, and I haven’t tried to sign anything or to build any installers.

Be aware that this uses the same profile directory as the installed release version of KeeWeb, so it is entirely possible to interfere with your “production” version if something goes wrong.
    
To use git, I found that I needed to add `package.json` and `package-lock.json` to `.git\info\exclude` — I think it was that Windows rewrites the line endings and makes it appear that they’ve changed. I also did `git update-index skip-worktree Gruntfile.js` to keep my changes to that file from being involved in anything I tried to commit. I honestly don’t remember why I used exclude for one and skip-worktree for the other.
